boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_nrf5340_cpuapp_common.dtsi

```diff
@@ -63,6 +83,13 @@
 
 &gpio0 {
 	status = "okay";
+
+	codec-interface {
+		gpio-hog;
+		gpios = <21 GPIO_ACTIVE_HIGH>;
```

Comment:
To not change too much from the way it was, I suggest having this as ACTIVE_LOW

Reply:
How does gpio-hog select this as low on boot? It seems that `output-low:` must be used somewhere when looking at the PR which introduced the hog functionality: https://github.com/zephyrproject-rtos/zephyr/pull/54110/files

Reply:
there's `output-low` (inactive, which means high level if using GPIO_ACTIVE_LOW) and `output-high` (active, which means low level if using GPIO_ACTIVE_LOW). I find naming a bit confusing, tbh. There should be `output-active` and `output-inactive`. If you want to keep `GPIO_ACTIVE_LOW` you need `output-high`.

Reply:
Ok, then this is fine.